### PR TITLE
Remove VoIP special stuff

### PR DESCRIPTION
**Pull request by drkitty**
**Resolves issue #945.**

*Requires a schema migration.*

### CODE REVIEW

**Comment by akeym** [comment]
Does this need a migration?

**Comment by drkitty** [comment]
Yes.

**Comment by murrown** [comment]
Shouldn't we remove the VoIP VRF from `initial_data`? I noticed that you removed it from the test fixtures but not from the regular fixtures. That being the case, does there need to be a data migration for networks that are currently using the VoIP VRF?

**Comment by akeym** [comment]
The existing VRF and members should not be removed/touched.

File: cyder/management/commands/dhcp_migrate.py [hunk 1]
```diff
@@ -98,8 +91,7 @@ def create_subnet(subnet_id, name, subnet, netmask, status, vlan_id):
                       "WHERE vlan_id = %s" % vlan_id):
         vlan_id, vlan_name, vlan_number = cursor.fetchone()
         vlan = Vlan.objects.get(name=vlan_name)
-    vrf = Vrf.objects.get(name=('VoIP' if network_str in voip_networks
-                                else 'Legacy'))
+    vrf = Vrf.objects.get(name='Legacy')
```
Comment (murrown) [inline cyder/management/commands/dhcp_migrate.py]: Just to be clear... there are no plans to run `dhcp_migrate` after merging this branch, right? So these changes are purely superficial. Is there any real reason to maintain `dhcp_migrate` and `dns_migrate` at this point? Even if we keep them updated, I doubt anyone will be testing them with any regularity.

Comment (akeym) [inline cyder/management/commands/dhcp_migrate.py]: we won't be running them again, that is correct.

**Comment by murrown** [comment]
alright it looks fine, I've also pushed the schema migration in another commit

**Pull request merged**